__tests__/test.js

```diff
@@ -128,40 +147,75 @@ test("delay and timeout", async () => {
 `);
 });
 
-test("loading error", async () => {
-  // Arrange
-  let LoadableMyComponent = Loadable({
-    loader: createLoader(400, null, new Error("test error")),
-    loading: MyLoadingComponent,
-  });
+describe("loading error", () => {
+  test("{throwOnError: true} (default)", async () => {
+    // Arrange
+    let LoadableMyComponent = Loadable({
+      loader: createLoader(400, null, new Error("test error")),
+      loading: MyLoadingComponent,
+    });
 
-  // Act
-  let component = renderer.create(<LoadableMyComponent prop="baz" />);
+    // Act
+    let component = renderer.create(
+      <ErrorBoundary>
+        <LoadableMyComponent prop="baz" />
+      </ErrorBoundary>
+    );
 
-  // Assert
-  // initial
-  expect(component.toJSON()).toMatchInlineSnapshot(`
+    // Assert
+    expect(component.toJSON()).toMatchInlineSnapshot(`
 <div>
   MyLoadingComponent 
   {"isLoading":true,"pastDelay":false,"timedOut":false,"error":null}
 </div>
 `);
-  await waitFor(200);
-  // loading
-  expect(component.toJSON()).toMatchInlineSnapshot(`
+    await waitFor(200);
+    expect(component.toJSON()).toMatchInlineSnapshot(`
 <div>
   MyLoadingComponent 
   {"isLoading":true,"pastDelay":true,"timedOut":false,"error":null}
 </div>
 `);
-  await waitFor(200);
-  // errored
-  expect(component.toJSON()).toMatchInlineSnapshot(`
+    await waitFor(200);
+    expect(component.toJSON()).toMatchInlineSnapshot(
+      `"ErrorBoundary: test error"`
+    );
```

Comment:
When using `throwOnError: true` (the default), the error boundary catches the error.